applications/CoSimulationApplication/co_simulation_api/co_sim_io_define.h

```diff
@@ -0,0 +1,48 @@
+// KRATOS  / ___|___/ ___|(_)_ __ ___  _   _| | __ _| |_(_) ___  _ ___
+//        | |   / _ \___ \| | '_ ` _ \| | | | |/ _` | __| |/ _ \| '_  |
+//        | |__| (_) |__) | | | | | | | |_| | | (_| | |_| | (_) | | | |
+//         \____\___/____/|_|_| |_| |_|\__,_|_|\__,_|\__|_|\___/|_| |_|
+//
+//  License:		 BSD License
+//                   license: CoSimulationApplication/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#ifndef KRATOS_CO_SIM_IO_DEFINE_H_INCLUDED
```

Comment:
@pooyan-dadvand in this file I check if the macros have been defined yet. If not I use default ones, otherwise they can be defined from outside

Reply:
:+1: